### ISSUE TITLE
Implementic support for embeddings

### ISSUE THREAD

**Issue by someshsingh22** [SforAiDl/decepticonlp#62]
This module needs to be planned well as we will need this for all our future implementations and will be required at all steps.
Currently, there are a lot of libraries available for this, common ones including Flair and Torchtext. We need a pipeline that will convert any textual input to vectors/matrices for further computation.

**Comment by Sharad24** [comment]
Torchtext has a module, no? Doesn't have transformers though, I think

**Comment by someshsingh22** [comment]
Yes, Yes, you can load transformers embeddings using torchtext easily but it's not in the library bas of now. So we will need a separate module that has torchtext's own as well as other methods.
